express-api/src/services/projects/projectsServices.ts

```diff
@@ -392,13 +392,13 @@ const updateProject = async (
       await AppDataSource.getRepository(ProjectStatusHistory).save({
         CreatedById: project.UpdatedById,
         ProjectId: project.Id,
-        WorkflowId: project.WorkflowId,
-        StatusId: project.StatusId,
+        WorkflowId: originalProject.WorkflowId,
+        StatusId: originalProject.StatusId, //I'm assuming that workflow and status should actually be from the now outdated version right?
```

Comment:
I had to check this against old PIMS, but yes, it seems to be the status it is leaving.